beta/nncf/tensorflow/quantization/algorithm.py

```diff
@@ -97,9 +103,18 @@ def _get_default_qconfig(self, constraints: QuantizationConstraints = None):
             qconfig = constraints.apply_constraints_to(qconfig)
         return qconfig
 
-    def _create_quantizer(self, name: str, qconfig: QuantizerConfig):
-        quantizer_cls = NNCF_QUANTIZATION_OPERATONS.get(qconfig.mode)
-        return quantizer_cls(name, qconfig)
+    def _get_half_range(self, qconfig):
+        if qconfig.num_bits == 8 and not self._disable_saturation_fix:
+            logger.warning('A saturation issue fix will be applied. '
```

Comment:
Why is it warning level but not info level?

Reply:
It is calque from PyTorch implementation. Should I change `warning` to `info`?

Reply:
Let's leave as is.